### DISCUSSION TITLE
hot reloading takes up to 5 seconds .  can it be any faster

### DISCUSSION THREAD

**Discussion by bobwatcherx** [reflex-dev/reflex#428] [Q&A]
I appreciate this project.  This is very nice .  small obstacle but a little annoying, namely hot reloading and seeing the changes can take up to 5-6 seconds.
if so maybe when reloading .  can add indicators such as page loading.  looks like it's processing.

  in contrast to next js which is very fast .  hopefully there will be an improvement later.  thanks

**Comment by Alek99** [comment]
There are ways it can be improved, we plan on only using the frontend packages you use in the future. Right now we are adding every package to bun despite if the user uses the feature or not. This should speed things up
